reducertodo/src/reducers/todo.js

```diff
@@ -0,0 +1,39 @@
+export const initialState = {
+  todos: [{ item: "Learn about reducers", completed: false, id: 3892987589 }]
+};
+
+export function reducer(state, action) {
+  switch (action.type) {
+    case "CHANGE_TODO":
+      return {
+        ...state,
+        item: action.payload
```

Comment:
inside your state, you currently only have the 1 property (the todos array), but here you're adding a item property.  If that is what you want, you should add the item property to your initialState